qiskit_optimization/runtime/qaoa_client.py

```diff
@@ -87,6 +116,12 @@ def __init__(
         self._initial_state = initial_state
         self._mixer = mixer
         self._reps = reps
+        self._use_swap_strategies = use_swap_strategies
```

Comment:
It would be nice if users could still change these attributes, maybe we can make them public attributes. For the `optimization_level` however we'd need a setter to raise the warning/error if the swap strategies are set.

Reply:
Good idea. Done here: 2598101